api/test/media_integration.py

```diff
@@ -14,6 +14,14 @@ def search(fixture):
     assert fixture["result_count"] > 0
 
 
+def search_all_excluded(media_path, excluded_source):
+    response = requests.get(
+        f"{API_URL}/v1/{media_path}?q=test&excluded_source={','.join(excluded_source)}"
+    )
+    data = json.loads(response.text)
+    assert data["result_count"] == 0
```

Comment:
Can you add a test for the validation error as well? Either as an integration test or as a unit test directly for the serializer?

Reply:
Added the test case in 8a97e14.